src/main/java/org/spongepowered/common/mixin/core/entity/MixinEntity.java

```diff
@@ -1183,4 +1185,9 @@ public Timing getTimingsHandler() {
         }
         return this.timing;
     }
+
+    @Override
+    public boolean hasGravity() {
+        return this.get(Keys.ENTITY_HAS_GRAVITY).get();
+    }
```

Comment:
See my comment on your API PR, i asked gabizou to confirm whether the method should return a boolean or a Value.